src/UI/FormTestResult.js

```diff
@@ -0,0 +1,551 @@
+        },
+        {
+          name: 'patient_date_of_birth',
+          value: passport.date_of_birth || '',
+        },
+        {
+          name: 'patient_gender_legal',
+          value: passport.sex || '',
+        },
+        {
+          name: 'patient_street_address',
+          value:
+            demographics.address.address_1 +
+              ' ' +
+              demographics.address.address_2 || '',
+        },
+        {
+          name: 'patient_city',
+          value: demographics.address.city || '',
+        },
+        {
+          name: 'patient_state',
+          value: demographics.address.state || '',
+        },
+        {
+          name: 'patient_postalcode',
+          value: demographics.address.zip_code || '',
+        },
+        {
+          name: 'patient_country',
+          value: demographics.address.country || '',
+        },
+        {
+          name: 'patient_phone',
+          value: demographics.phone || '',
+        },
+        {
+          name: 'sending_facility',
+          value: form.get('sending_facility') || '',
+        },
+        {
+          name: 'visit_location',
+          value: form.get('visit_location') || '',
+        },
+        {
+          name: 'observation_date_time',
+          value: form.get('observation_date_time') || '',
+        },
+        {
+          name: 'lab_specimen_collected_date',
+          value: form.get('lab_specimen_collected_date') || '',
+        },
+        {
+          name: 'result_status',
+          value: form.get('result_status') || '',
+        },
+        {
+          name: 'lab_coding_qualifier',
+          value: form.get('lab_coding_qualifier') || '',
+        },
+        {
+          name: 'lab_code',
+          value: form.get('lab_code') || '',
+        },
+        {
+          name: 'lab_description',
+          value: form.get('lab_description') || '',
+        },
+        {
+          name: 'lab_order_id',
+          value: form.get('lab_order_id') || '',
+        },
+        {
+          name: 'ordering_facility_address',
+          value: form.get('ordering_facility_address') || '',
+        },
+        {
+          name: 'ordering_facility_name',
+          value: form.get('ordering_facility_name') || '',
+        },
+        {
+          name: 'date_time_of_message',
+          value: form.get('date_time_of_message') || '',
+        },
+        {
+          name: 'normality',
+          value: form.get('normality') || '',
+        },
+        {
+          name: 'result',
+          value: form.get('result') || '',
+        },
+        {
+          name: 'comment',
+          value: form.get('comment') || '',
+        },
+        {
+          name: 'performing_lab',
+          value: form.get('performing_lab') || '',
+        },
+      ]
+    }
+    let newCredential = {
+      connectionID: props.contactSelected.Connections[0].connection_id,
+      schemaID: 'X2JpGAqC7ZFY4hwKG6kLw9:2:Covid_19_Lab_Result:1.5',
```

Comment:
We got the wrong schema on this one, do we have a good branch to cherry pick form?

Reply:
This form isn't in use, we'll get rid of it.